src/System.CommandLine/Builder/CommandLineBuilderExtensions.cs

```diff
@@ -643,6 +624,7 @@ public static CommandLineBuilder UseHelp(this CommandLineBuilder builder, int? m
             builder.VersionOption = versionOption;
             builder.Command.AddOption(versionOption);
 
+#if false
```

Comment:
```suggestion
```

Reply:
Actually I misunderstood, this is disabling the code, right?

Do we want to keep it that way or remove the whole block?

Reply:
I would propose removal of the whole block